pyemd/emd.pyx

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
Thanks! I know, but on ArchLinux for instance python3 is python. Python
might also be in another path...The shebang is system-dependent.

Plus it is not useful unless you want to make the code executable, which is
not what we do here. I can put it back, but I don't think it is necessary
here!
On 24 May 2015 18:12, "Will Mayner" <notifications@github.com> wrote:

> In pyemd/emd.pyx
> <https://github.com/wmayner/pyemd/pull/3#discussion_r30953516>:
>
> > @@ -1,4 +1,3 @@
> > -#!/usr/bin/env python3
>
> This line actually isn't useless; see
> http://stackoverflow.com/a/2429515/1085344.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wmayner/pyemd/pull/3/files#r30953516>.
>